CP03ISSM/CP03ISSM_R00003_ingest.csv

```diff
@@ -1,43 +1,43 @@
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl27/syslog/*.syslog.log,CP03ISSM-RID27-00-DCLENG000,recovered_host,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl27/OPTAA_sn123/*.optaa1*.log,CP03ISSM-RID27-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl27/FLORT_sn1324/*.flort.log,CP03ISSM-RID27-02-FLORTD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl27/CTDBP_sn16-50109/*ctdbp1*.log,CP03ISSM-RID27-03-CTDBPC000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl27/CTDBP_sn16-50109/SBE*.hex,CP03ISSM-RID27-03-CTDBPC000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl27/DOSTA_sn508/*dosta1*.log,CP03ISSM-RID27-04-DOSTAD000,recovered_host,Available,
+#mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/irid2shore/cpm_status*.txt,CP03ISSM-SBC11-00-CPMENG000,recovered_host,Not Expected,It was not routinely captured.
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl11/syslog/*.syslog.log,CP03ISSM-SBD11-00-DCLENG000,recovered_host,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl11/MOPAK_sn/*.mopak.log,CP03ISSM-SBD11-01-MOPAK0000,recovered_host,Not Expected,It was not routinely captured.
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl11/hyd1/*.hyd1.log,CP03ISSM-SBD11-02-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl11/METBK-Log_snLGR023/*.metbk*.log,CP03ISSM-SBD11-06-METBKA000,recovered_host,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl12/syslog/*.syslog.log,CP03ISSM-SBD12-00-DCLENG000,recovered_host,Available,
+#mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl12/Hydrogen_sn/*.hyd2.log,CP03ISSM-SBD12-03-HYDGN0000,recovered_host,Not Expected,It was not routinely captured.
+mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl12/PCO2A_sn34-260-50A/*pco2a.log,CP03ISSM-SBD12-04-PCO2AA000,recovered_host,Available,
+#mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP03ISSM/R00003/cg_data/dcl37/ZPLSC_sn55100/DATA/*/*.01A,CP03ISSM-MFD37-07-ZPLSCC000,recovered_inst,Available,Test parser
```

Comment:
Commented out this line because we cannot correctly ingest ZPLSC recovered instrument data yet